cpp/modmesh/mesh/StaticMesh.hpp

```diff
@@ -309,6 +309,10 @@ class StaticMesh
     uint_type nedge() const { return static_cast<uint_type>(m_ednds.shape(0)); }
     size_t nbcs() const { return m_bcs.size(); }
 
+    void set_ncell(uint_type ncell) { m_ncell = ncell; }
```

Comment:
It is intentional to avoid setters of the `n*` member data.  These setters should not be added.

Reply:
I got it! I'll remove them.